gcp/README.md

```diff
@@ -138,11 +138,14 @@ Deploying to `stg` verifies that the gpii-infra code that worked to create a `de
 
 Because `stg` emulates production, it will (in the future) allow us to run realistic end-to-end tests before deploying to `prd`.
 
+The `stg` environment is almost the same environment as `prd`, this fact makes `stg` the proper environment to test some recovering procedures that the ops team must perform periodically. 
```

Comment:
First part of this sentence duplicates the statement on line 139. Maybe combine them?